src/test/mochitest/browser_dbg-breakpoints-actions.js

```diff
@@ -14,8 +13,58 @@ add_task(async function() {
   await waitForSelectedSource(dbg, "simple2");
 
   await addBreakpoint(dbg, "simple2", 3);
-  await removeBreakpoint(dbg);
+
+  openFirstBreakpointContextMenu(dbg)
+  // select "Remove breakpoint"
+  selectMenuItem(dbg, 1);
 
   await waitForState(dbg, state => dbg.selectors.getBreakpointCount(state) === 0);
-  ok("successfully removed the breakpoint")
+  ok("successfully removed the breakpoint");
+});
+
+// Tests "disable others", "enable others" and "remove others" context actions
+add_task(async function() {
+  const dbg = await initDebugger("doc-scripts.html");
+  await selectSource(dbg, "simple1");
+  await waitForSelectedSource(dbg, "simple1");
+
+  await addBreakpoint(dbg, "simple1", 1);
+  await addBreakpoint(dbg, "simple1", 4);
+  await addBreakpoint(dbg, "simple1", 5);
+  await addBreakpoint(dbg, "simple1", 6);
+
+  openFirstBreakpointContextMenu(dbg)
+  // select "Disable Others"
+  selectMenuItem(dbg, 7);
+  await waitForState(dbg, state =>
+    dbg.selectors.getBreakpointsList(state)
+      .every(bp => (bp.location.line !== 1) === bp.disabled)
+  );
+  ok("breakpoint at 1 is the only enabled breakpoint");
+
+  openFirstBreakpointContextMenu(dbg)
+  // select "Disable All"
+  selectMenuItem(dbg, 9);
+  await waitForState(dbg, state =>
+    dbg.selectors.getBreakpointsList(state).every(bp => bp.disabled)
+  );
+  ok("all breakpoints are disabled")
+
+  openFirstBreakpointContextMenu(dbg)
+  // select "Enable Others"
+  selectMenuItem(dbg, 3);
+  await waitForState(dbg, state =>
+    dbg.selectors.getBreakpointsList(state)
+      .every(bp => (bp.location.line === 1) === bp.disabled)
```

Comment:
okay, this is the craziest thing I've seen in awhile, but it's a test so lets go crazy